src/main.rs

```diff
@@ -191,7 +195,18 @@ fn main() -> Result<(), lexopt::Error> {
     }
 
     let config = {
-        if let Ok(mut f) = File::options().read(true).open(CONFIG_FILE_NAME) {
+        let open_config = |file_name| File::options().read(true).open(file_name);
+        let config = open_config(CONFIG_FILE_NAME);
+        let dot_config_file_name = format!(".{CONFIG_FILE_NAME}");
+        let dot_config = open_config(&dot_config_file_name);
+        if config.is_ok() && dot_config.is_ok() {
+            eprintln!(
+                "Warning! Both \"{first}\" and \"{second}\" are present. Using \"{first}\".",
+                first = CONFIG_FILE_NAME,
+                second = dot_config_file_name
+            );
+        }
+        if let Ok(mut f) = config.or(dot_config) {
             let mut buf = String::default();
             f.read_to_string(&mut buf).unwrap_or_else(|err| {
                 panic!("Failed to read config file {CONFIG_FILE_NAME:?}: {err}")
```

Comment:
I feel like now, this can be confusing, can I ask you to fix it by replacing this line
`                panic!("Failed to read config file {CONFIG_FILE_NAME:?}: {err}")`
with 
`                panic!("Failed to read config file {CONFIG_FILE_NAME:?} (or/then .{CONFIG_FILE_NAME:?}): {err}")`
Or something else if you have a better idea?

If you're tired of this PR and want to merge already no problem, I can take care of it

Reply:
What about next error:

```rs
Config::from_toml(&buf).unwrap_or_else(|e| panic!("Config file invalid: {e}.\nYou'll maybe have to delete it and use -C to create a default config file."))
```

Error about reading config file should include file name, but TOML parser error shouldn't?

Reply:
That's also getting out of date, I didn't notice it. It should include the config file used, 

maybe the most simple solution is to, in verbose mode, print which config file is used? But I think it would be nicer if each error contained the name of the file

Reply:
We don't even have a verbose mode yet. Ok, I'll update this message too. Everything else can be changed after this PR.

Reply:
We do have a verbose mode, it's when verbose is true, did you rebase work on master?

Reply:
Ohhhh, I checked on the old binary while being on the latest changes. Yes, there is a verbose option. The description says `increase verbosity for non errors`. Pretty sure that `panic!()` is an error.

Reply:
I meant we could have the error just complain and the verbose option always print the config file used (even on no error)

Reply:
Ahh (like an init logs), no that's definitely a separate feature.

Reply:
:+1: